src/views/admin/oneorder.ejs

```diff
@@ -9,132 +9,80 @@
               <div class="card-header py-3">
                 <div class="list-group-item d-flex justify-content-between align-items-center border-0 px-0 pb-0">
                   <h5>Order</h5>
-                  <span class="badge bg-warning text-dark">Pending</span>
+                  <% let status=order.status %>
+                    <%if (status=='pending' ) { %>
+                      <span class="badge bg-warning text-dark">Pending</span>
+                      <% } else if (status=='done' ) { %>
+                        <span class="badge bg-success">Done</span>
+                        <% } %>
```

Comment:
1. status nie wyświtle się, ponieważ w bazie danych jest reprezentowany jako numer.
0 - pending
1 - done
2. Może ja zle to napisałem ale status jest z dużej litery 
Mój quickfix:
`
<% let status=order.Status %>
                    <%if (status===0 ) { %>
                      <span class="badge bg-warning text-dark">Pending</span>
                      <% } else if (status===1 ) { %>
                        <span class="badge bg-success">Done</span>
                        <% } %>`